module-apps/application-phonebook/widgets/PhonebookListView.hpp

```diff
@@ -14,16 +14,20 @@ namespace gui
 
     class PhonebookListView : public ListView
     {
-        UTF8 labelMark           = "";
-        UTF8 previousLabelMark   = "";
-        bool previousItemIsLabel = false;
-
+      public:
+        PhonebookListView(Item *parent,
+                          std::uint32_t x,
+                          std::uint32_t y,
+                          std::uint32_t w,
+                          std::uint32_t h,
+                          std::shared_ptr<ListItemProvider> prov);
+
+      private:
         void addItemsOnPage() override;
         void addLabelMarker(PhonebookItem *item);
+        unsigned getSlotsLeft();
```

Comment:
Maybe `uint32_t` or `size_t` to be more explicit?

Reply:
Changed to `std::size_t` ;)